package-lock.json

```diff
@@ -1,7 +1,2505 @@
 {
   "name": "tech-docs-gem",
+  "lockfileVersion": 2,
```

Comment:
This is currently part of 469f2d492b09d4383dbee905e1eb819e865a7508 but has nothing to do with the rest of the commit. I think this is just because you're running npm v7 locally and so running `npm install` generates a new v2 lockfile? Suggest this change is made in a separate PR.

Reply:
Think this was related to installing inert (first tried through npm) - reverted that.